server/routes/visit.test.ts

```diff
@@ -583,61 +583,45 @@ describe('GET /visit/:reference/cancel', () => {
         expect($('label[for="cancel"]').text().trim()).toBe('Visitor cancelled')
         expect($('[data-test="administrative_error"]').attr('value')).toBe('ADMINISTRATIVE_ERROR')
         expect($('label[for="cancel-4"]').text().trim()).toBe('Administrative error')
+
+        expect($('input[name="method"]').eq(0).prop('value')).toBe('PHONE')
+        expect($('input[name="method"]').eq(1).prop('value')).toBe('WEBSITE')
+        expect($('input[name="method"]').eq(2).prop('value')).toBe('EMAIL')
+        expect($('input[name="method"]').eq(3).prop('value')).toBe('IN_PERSON')
+        expect($('input[name="method"]:checked').length).toBe(0)
+
         expect($('[data-test="cancel-booking"]').length).toBe(1)
       })
   })
 
-  it('should render the cancellation reasons page with no selection validation error', () => {
```

Comment:
Slimmed this down a bit to just check all the validation errors and `formValues` population in one test